src/folsom_cowboy_app.erl

```diff
@@ -33,19 +33,24 @@
 -define(APP, folsom_cowboy).
 
 start(_Type, _Args) ->
-    Dispatch = cowboy_router:compile(env(dispatch)),
+    {ok, _} = application:ensure_all_started(cowboy),
```

Comment:
I was hoping for plug and play, really, so you could just include the application and it'd automatically set up its own endpoints, separate from those you're already providing.